src/engine/x86-64/X86_64Interpreter.v3

```diff
@@ -3249,6 +3273,11 @@ class X86_64InterpreterGen(ic: X86_64InterpreterCode, w: DataWriter) {
 			asm.movq_r_m(r_dispatch, masm.absPointer(offsets.Interpreter_dispatchTable));
 		}
 	}
+	// called after switching to a new stack
```

Comment:
I think I prefer this to remain inline; otherwise it seems to imply there is something special about stack switches, which doesn't seem to be the case.